framework/src/transfers/MultiAppFieldTransfer.C

```diff
@@ -48,6 +48,9 @@ MultiAppFieldTransfer::MultiAppFieldTransfer(const InputParameters & parameters)
         getParam<std::vector<PostprocessorName>>("to_postprocessors_to_be_preserved")),
     _use_nearestpoint_pps(false)
 {
+  if (_directions.size() != 1)
+    paramError("direction", "This transfer is only unidirectional");
+
```

Comment:
I'm not sure if this is the best way to go considering it has to be added for _everything_ at this point

Reply:
I guess so for now - what do you think @permcody

Reply:
I mean - we could add a thing where the Transfer registers which directions are valid...

Reply:
Nah, Right now all directions are valid, it's the multiplicity that has to be coded for at this point. I don't think saying the transfer can support both directions simultaneously is all that clean. I'd suggest that we just go with the param error for now as it won't be that much work to make all the existing transfers support simultaneous directions.